base-image/Dockerfile

```diff
@@ -4,10 +4,14 @@ FROM ubuntu:20.04
 LABEL org.opencontainers.image.source=https://github.com/pangeo-data/pangeo-docker-images
 
 # Run this section as root
-# try to keep conda version in sync with repo2docker?
-# https://github.com/jupyterhub/repo2docker/blob/297c7955d208f3595c4d5bc2b079953199bb1f35/repo2docker/buildpacks/conda/install-miniforge.bash#L8-L9# ========================
-ENV CONDA_VERSION=4.10.3-7 \
-    MAMBA_VERSION=0.17 \
+# CONDA_VERSION: A miniforge release version found at
+#                https://github.com/conda-forge/miniforge/releases specified to
+#                either just "major" or "major.minor", but always excluding
+#                ".patch-build".
+# MAMBA_VERSION: A conda-forge package version found at
+#                https://anaconda.org/conda-forge/mamba
+ENV CONDA_VERSION=4.11 \
+    MAMBA_VERSION=0.20 \
```

Comment:
@consideRatio I'm curious if you need mamba=0.20, an alternative simplification could be to just use CONDA_VERSION and pull the newer "mambaforge" installers that did not previously exist (https://github.com/conda-forge/miniforge/releases/download/4.11.0-0/Mambaforge-4.11.0-0-Linux-x86_64.sh). It's not clear to me what mamba version is packaged within though...

Reply:
Going straight for a mambaforge installer makes perfect sense to me.

I can do that as part of this PR if you think such change is okay. Doing so, would it be okay if i removed the environment variables CONDA_VERSION / MAMBA_VERSION as well and added MAMBAFORGE_VERSION?

Reply:
yes sounds great! as far as I can tell, it's the same setup just with mamba included by default

Reply:
Going for it!

Reply:
```
    && conda install -y -c conda-forge mamba=${MAMBA_VERSION} \
    && mamba clean -afy \
    && find ${CONDA_DIR} -follow -type f -name '*.a' -delete \
    && find ${CONDA_DIR} -follow -type f -name '*.pyc' -delete
```

Do you have ideas on if this stuff needs to be cleaned up still?

Reply:
yes, i think all 4 lines can be removed now

Reply:
I've concluded those cleanup commands still has work to do, at least the last two.

Reply:
interesting! thanks for checking